src/Docfx.App/RunBuild.cs

```diff
@@ -22,6 +24,12 @@ public static string Exec(BuildJsonConfig config, BuildOptions options, string c
 
         EnvironmentContext.SetGitFeaturesDisabled(config.DisableGitFeatures);
         EnvironmentContext.SetBaseDirectory(Path.GetFullPath(string.IsNullOrEmpty(configDirectory) ? Directory.GetCurrentDirectory() : configDirectory));
+
+        if (!config.DisableGitFeatures)
+        {
+            CheckGitCommandExists();
```

Comment:
nit: consider use `Task.Run` so we don't need to explicitly catch here:
```suggestion
            Task.Run(() => GitUtility.ExistGitCommand.Value);
```

Reply:
Refactor code  as suggested that using `Task.Run`.

`Task.Run` without await/Wait should not throw any Exception (because thrown exception can be handled at `TaskScheduler.UnobservedTaskException` only) So I've moved try-catch logics to Lazy initializer.